sources/packages/backend/apps/api/src/route-controllers/student-account-applications/models/student-account-application.dto.ts

```diff
@@ -37,6 +37,4 @@ export class StudentAccountApplicationApprovalAPIInDTO extends CreateStudentAPII
   email: string;
   @Allow()
   dateOfBirth: string;
-  @Allow()
-  gender: string;
```

Comment:
Why was this removed from the Ministry approval? The Ministry should be able to change any information and create the student. This information was already coming from the form, not from BCSC. Can you please elaborate on why was it removed or what I am missing here?